DNA-RNA_classes.py

```diff
@@ -0,0 +1,127 @@
+class Rna:
+    """A Rna object contains a RNA sequence.
+    Attribute: seq - the sequence itself"""
+
+    def __init__(self, seq):
+        if isinstance(seq, str):
+            self.seq = seq.upper()
+        else:
+            raise TypeError("Please, pass a string argument")
+
+    def __len__(self):
+        """Returns the length of the sequence"""
+        return len(self.seq)
+
+    def __add__(self, other):
+        """
+        Add another sequence or a string to the sequence
+        :param other: python string or Dna object
+        :return: new Dna object with appended sequence
+        """
+        if not isinstance(other, Rna):
+            if isinstance(other, str):
+                return Rna(self.seq + other)
+            else:
+                raise TypeError("Other sequence should be a string")
+        else:
+            return Rna(self.seq + other.seq)
```

Comment:
Меньше вложенность кода, проще читать
```suggestion
        if isinstance(other, Rna):
            return Rna(self.seq + other.seq)
        if isinstance(other, str):
            return Rna(self.seq + other)
        raise TypeError("Other sequence should be a string")
            
```